certified-connectors/Adobe Experience Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,431 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Adobe Experience Manager",
+    "description": "Adobe Experience Manager APIs allow access to Adobe Experience Manager Forms related services(e.g. Communications and more). This integration will also allow flows to be triggered when a form is submitted through Adobe Experience Manager Forms.",
```

Comment:
@syamavarm Please remove the word API from the description. The description should meet our [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
I have changed it.  But I have a question , more for my understanding. paconn validate did not catch this. May be, this is at "warn" level?

Reply:
Thank you so much @syamavarm for the fix. Please see out docs for more details. 
https://learn.microsoft.com/en-gb/connectors/custom-connectors/connector-string-guidance#connector-info
https://learn.microsoft.com/en-gb/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector